MAINTAINERS

```diff
@@ -1,8 +1,58 @@
-Solomon Hykes <solomon@docker.com> (@shykes)
-Olivier Gambier <olivier@docker.com> (@dmp42)
-Stephen Day <stephen.day@docker.com> (@stevvooe)
-Derek McGowan <derek@mcgstyle.net> (@dmcgowan)
-Richard Scothern <richard.scothern@gmail.com> (@richardscothern)
-Aaron Lehmann <aaron.lehmann@docker.com> (@aaronlehmann)
+# Distribution maintainers file
+#
+# This file describes who runs the docker/distribution project and how.
+# This is a living document - if you see something out of date or missing, speak up!
+#
+# It is structured to be consumable by both humans and programs.
+# To extract its contents programmatically, use any TOML-compliant parser.
+#
+# This file is compiled into the MAINTAINERS file in docker/opensource.
+#
+[Org]
+	[Org."Core maintainers"]
```

Comment:
You are the "core maintainers" of your repo.






On Tue, Dec 8, 2015 at 11:43 AM -0800, "Sebastiaan van Stijn" <notifications@github.com> wrote:












In MAINTAINERS:
> -Stephen Day <stephen.day@docker.com> (@stevvooe)
> -Derek McGowan <derek@mcgstyle.net> (@dmcgowan)
> -Richard Scothern <richard.scothern@gmail.com> (@richardscothern)
> -Aaron Lehmann <aaron.lehmann@docker.com> (@aaronlehmann)
> +# Distribution maintainers file
> +#
> +# This file describes who runs the docker/distribution project and how.
> +# This is a living document - if you see something out of date or missing, speak up!
> +#
> +# It is structured to be consumable by both humans and programs.
> +# To extract its contents programmatically, use any TOML-compliant parser.
> +#
> +# This file is compiled into the MAINTAINERS file in docker/opensource.
> +#
> +[Org]
> +	[Org."Core maintainers"]



Current version can be found here; this is what it looks like; https://github.com/docker/opensource/blob/master/MAINTAINERS#L225



—
Reply to this email directly or view it on GitHub.